extension/chrome/elements/compose.ts

```diff
@@ -2,21 +2,22 @@
 
 'use strict';
 
-import { Store, AccountStoreExtension, Scopes, AccountStore } from '../../js/common/platform/store.js';
-import { Ui } from '../../js/common/browser/ui.js';
-import { Composer } from './composer/composer.js';
-import { BrowserMsg } from '../../js/common/browser/browser-msg.js';
-import { openpgp } from '../../js/common/core/pgp.js';
-import { ReplyParams, EmailProviderInterface } from '../../js/common/api/email_provider/email_provider_api.js';
+import { AccountStore, AccountStoreExtension, Scopes, Store } from '../../js/common/platform/store.js';
+import { EmailProviderInterface, ReplyParams } from '../../js/common/api/email_provider/email_provider_api.js';
+
```

Comment:
It seemed to have produced unnecessary empty lines throughout the PR. Otherwise it looks good. Thanks!